commands/deploy.go

```diff
@@ -162,14 +169,35 @@ func (p *deploycmd) deploySingle(c *cli.Context, appName string, appf *common.Ap
 		dir = filepath.Join(wd, path)
 	}
 
-	fmt.Println("dir: ", dir)
-
 	err := os.Chdir(dir)
 	if err != nil {
 		return err
 	}
 	defer os.Chdir(wd)
 
+	ffV, err := common.ReadInFuncFile()
+	version := common.GetFuncYamlVersion(ffV)
+	if version == common.LatestYamlVersion {
```

Comment:
again switch. The reason is to keep the function return statements close and understandable.